src/broker.js

```diff
@@ -58,6 +62,14 @@ module.exports = async (config, startUpLoggingEnabled = false) => {
         log('Cassandra ### NEW SCHEMA ### is disabled')
     }
 
+    // Ethereum authentication
+    const wallet = await ethereumAuthenticate.authenticateFromConfig(config.ethereum, log)
```

Comment:
You could just return the address directly from `authenticateFromConfig` for simplicity?